typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -140,8 +140,23 @@ const fontInfo = {
 };
 
 export default class Axes2DLayer extends Layer<Axes2DLayerProps> {
-    initializeState(context: LayerContext): void {
-        const { gl } = context;
+    shouldUpdateState() {
+        return true;
```

Comment:
should be `changeFlags.somethingChanged`?

Reply:
I want to uodate the state for each rendering

Reply:
`somethingChanged` includes viewport has changed. Shouldn't that cover it?

Reply:
fixed